Xwt.Gtk.Mac/GtkMacOpenFileDialogBackend.cs

```diff
@@ -46,7 +46,7 @@ public void Initialize (System.Collections.Generic.IEnumerable<FileDialogFilter>
 			if (!string.IsNullOrEmpty (initialFileName))
 				this.DirectoryUrl = new NSUrl (initialFileName, true);
 
-			this.Prompt = "Select File" + (multiselect ? "s" : "");
+			this.Prompt = Application.TranslationCatalog.GetPluralString("Select File", "Select Files", multiselect ? 2 : 1);
```

Comment:
we could use a NSOpenSavePanelDelegate to track selection changes (same in https://github.com/sevoku/xwt/blob/add-translation-support/Xwt.XamMac/Xwt.Mac/FileDialogBackend.cs#L58). And we could also add some logic to Gtk/Wpf.

But actually I don't really like the plurals here, cause alsmost no other App does this. I think something more generic like "Select" (without "File"/"Folder") would fit better. Its even worse in Gtk (https://github.com/sevoku/xwt/blob/add-translation-support/Xwt.Gtk/Xwt.GtkBackend/FileDialogBackend.cs#L65-L81) where we use generic "non"-plural stock ids for other commands and folder selection/creation is always "Select Folder".